src/local-history/local-history-manager.ts

```diff
@@ -449,4 +451,21 @@ export class LocalHistoryManager {
             .update(str, 'utf8')
             .digest('hex');
     }
+
+    /**
+     * Exclude the revisions of sources which follows the glob pattern. 
+     * @param document Represent the active text document.
+     */
+    private excludeFiles(document: vscode.TextDocument): boolean {
```

Comment:
minor: since the method has been updated to check if the current document matches any of the exclude glob patterns and the fact that it returns a `boolean` I think we should update the naming of the method:


```ts
isFileExcluded(document: vscode.TextDocument): boolean { ... }
```


or

```ts
shouldExcludeFile(document: vscode.TextDocument): boolean { ... }
```